src/main/java/org/terasology/logic/health/event/DoHealEvent.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.logic.health.event;
+
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.Event;
+
+/**
+ * Send this event to an entity to heal it.
```

Comment:
"to heal it ..." - fully? Or could a negative number be hurtful instead? Maybe that's covered in the System

Reply:
A negative value is supposed to cause damage, but not working at present. Will update the Javadoc when the issue is solved.